Common/Orders/OrderEvent.cs

```diff
@@ -111,6 +121,25 @@ public OrderEvent(Order order, string message = "")
         }
 
         /// <summary>
+        /// Helper Constructor for TradeBuilder tests
+        /// </summary>
+        /// <param name="orderId">The Id of the order</param>
+        /// <param name="symbol">Asset Symbol</param>
+        /// <param name="time">Date/time of this event</param>
+        /// <param name="fillPrice">Fill price</param>
+        /// <param name="fillQuantity">Fill quantity</param>
+        /// <param name="orderFee">The order fee</param>
+        public OrderEvent(int orderId, string symbol, DateTime time, decimal fillPrice, int fillQuantity, decimal orderFee)
```

Comment:
If possible let's merge the two test helper constructors, that sounds great. As for the other `OrderEvent(Order)` constructor, I think we'll need to add a utcTime parameter and an orderFee parameter. I'm ok with these being addressed in a separate PR.